crypton-conduit.cabal

```diff
@@ -43,7 +43,6 @@ test-suite crypton-conduit-test
   build-depends:       base
                      , bytestring
                      , conduit
```

Comment:
I think a Cabal flag is needed (`conduit-1-3-0`, say), True by default. If True, uses `conduit >= 1.3.0` (only). If False, uses `conduit < 1.3.0` and `conduit-combinators`.